.github/workflows/coverage.yaml

```diff
@@ -15,32 +15,33 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.11
-          cache: 'pip'
-          cache-dependency-path: 'eodhp-utils/*.txt'
+          python-version: 3.12
+          cache: "pip"
+          cache-dependency-path: |
+            requirements.txt
+            requirements-dev.txt
 
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -r requirements-dev.txt
-          pip install -r requirements.txt
+          pip install -r requirements-dev.txt -r requirements.txt
 
       - name: Code Coverage
         run: |
-          coverage run -m pytest
-          coverage report
+          coverage run -m pytest -vv
+          coverage report --omit=tests/\*
           COVERAGE_PERCENT=$(coverage report | awk 'END{print $4}')
           COVERAGE=${COVERAGE_PERCENT%\%}
           echo "COVERAGE=$COVERAGE" >> "$GITHUB_ENV"
 
       - name: Coverage Check
         run: |
-          if (( $(echo "$COVERAGE < 100" | awk '{print ($1 < 100)}' ) )); then
+          if (( $(echo "$COVERAGE < 80" | awk '{print ($1 < 80)}' ) )); then
```

Comment:
Only 80?

Reply:
80% is higher than most people aim for. I'd worry that being too stringent would lead people to chase the metric by covering lines rather than writing good tests.